vm/src/builtins/code.rs

```diff
@@ -5,15 +5,39 @@
 use super::{PyStrRef, PyTupleRef, PyType, PyTypeRef};
 use crate::{
     builtins::PyStrInterned,
-    bytecode::{self, BorrowedConstant, Constant, ConstantBag},
+    bytecode::{self, BorrowedConstant, CodeFlags, Constant, ConstantBag},
     class::{PyClassImpl, StaticType},
     convert::ToPyObject,
-    function::FuncArgs,
+    function::{FuncArgs, OptionalArg},
     AsObject, Context, Py, PyObject, PyObjectRef, PyPayload, PyRef, PyResult, VirtualMachine,
 };
 use num_traits::Zero;
 use std::{borrow::Borrow, fmt, ops::Deref};
 
+#[derive(FromArgs)]
+pub struct ReplaceArgs {
+    #[pyarg(any, optional)]
```

Comment:
```suggestion
    #[pyarg(named, optional)]
```
for every lines. because it doesn't take positional argument